content/_data/changelogs/weekly.yml

```diff
@@ -982,6 +982,29 @@
       pull: 2956
     # pull: 2971 not notable enough
     # pull: 2968 not notable enough
+- version: "2.75"
+  date: 2017-08-20
+  changes:
+    - type: rfe
+      message: >
+        Update Windows service wrapper from 2.1.0 to 2.1.2 and Windows Agent Installer from 1.9 to 1.9.1.
```

Comment:
So you do not reference issues at all, right? 
It's mentioned as "RFE", but effectively there are 3 bugfixes. All of them may impact Jenkins users (2 - OOTB, 1 - if the custom stop executable is defined)